src/main/java/kitchenpos/menu/domain/Menu.java

```diff
@@ -0,0 +1,70 @@
+package kitchenpos.menu.domain;
+
+import kitchenpos.menugroup.domain.MenuGroup;
+import kitchenpos.menu.exception.IllegalMenuPriceException;
+import kitchenpos.menugroup.exception.NoMenuGroupException;
+import kitchenpos.product.domain.Price;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@Entity
+public class Menu {
+    public static final int MIN_MENU_PRICE = 0;
+    @Id
```

Comment:
```suggestion
    public static final int MIN_MENU_PRICE = 0;
    
    @Id
```

클래스 변수 사이에 공백을 추가한다면 가독성이 상승합니다!